eng/targets/Imports.targets

```diff
@@ -367,6 +367,15 @@
     </ItemGroup>
   </Target>
   
+  <!-- TODO: Remove once we consume an SDK with the fix from https://github.com/dotnet/wpf/pull/6799 -->
```

Comment:
Nit:  can you match the general format for a work around comment as used in the rest of the file 

```suggestion
  <!-- Workaround for https://github.com/dotnet/wpf/pull/6799 -->
```